src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/ApprenticeshipDetailsRequestViewModel.cs

```diff
@@ -34,21 +35,17 @@ public class ApprenticeshipDetailsRequestViewModel : IAuthorizationContextModel
         public bool HasPendingChangeOfProviderRequest { get; set; }
         public Party? PendingChangeOfProviderRequestWithParty { get; set; }
         public string HashedNewApprenticeshipId { get; set; }
-        public bool IsContinuation { get; set; }
-        public string HashedPreviousApprenticeshipId { get; set; }
         public bool ShowChangeTrainingProviderLink => ((ApprenticeshipStatus == ApprenticeshipStatus.Stopped ||
                                                        ApprenticeshipStatus == ApprenticeshipStatus.Paused ||
                                                        ApprenticeshipStatus == ApprenticeshipStatus.Live ||
                                                        ApprenticeshipStatus == ApprenticeshipStatus.WaitingToStart) &&
                                                        !HasPendingChangeOfProviderRequest &&
                                                        !HasPendingChangeOfEmployerRequest &&
-                                                       !(HasApprovedChangeOfEmployerRequest && !IsContinuation) &&
```

Comment:
has this condition been correctly remove, meaning it was being applied incorrectly previously?

Reply:
Yes - the next line is using HashedNewApprenticeshipId which is now correctly obtained for both ChangeOfEmployer and ChangeOfProvider - this was a bug identified in the previous migration work and an AC was added to this ticket to fix it as the testing in in the same area